### PR TITLE
Homepage: More focus on open source

### PR DESCRIPTION
**Pull request by ewels**
Updates on the homepage, see highlighted areas:

CleanShot 2024-09-12 at 23 36 30@2x

CleanShot 2024-09-12 at 23 37 35@2x

### CODE REVIEW

File: sites/main-site/src/pages/index.astro [hunk 1]
```diff
@@ -127,6 +133,11 @@ import AnnouncementBanner from '@components/navbar/AnnouncementBanner.astro';
         </div>
     </div>
 
+    <div class="container pb-5 text-center lead" id="features">
+        <p>nf-core pipelines are <strong>community owned</strong>.</p>
+        <p>Everyone is welcome to contribute, and maintainers can change over time.</p>
```
Comment (christopher-hakkaart) [inline sites/main-site/src/pages/index.astro:138]:
```suggestion
        <p>Everyone is welcome to use the project and contribute to its growth.</p>
```

**Review by christopher-hakkaart — COMMENTED** [review]
Minor suggestion - feel free to ignore

File: sites/main-site/src/pages/index.astro (hunk 1 above)
Comment (ewels) [reply sites/main-site/src/pages/index.astro:138]: Reads better, but changes the meaning a bit. I was trying to follow on from the "community owned" bit to emphasise that people can come and go..

Comment (christopher-hakkaart) [reply sites/main-site/src/pages/index.astro:138]: The word "maintainers" won't mean much to someone new to the community. Maybe something like "Everyone is welcome to use, contribute to, and help maintain this project over time"

Comment (ewels) [reply sites/main-site/src/pages/index.astro:138]: How about this?
```suggestion
        <p>Everyone is welcome to use, contribute to, and help maintain pipelines.</p>
```

File: sites/main-site/src/pages/index.astro [hunk 2]
```diff
@@ -25,7 +25,8 @@ import AnnouncementBanner from '@components/navbar/AnnouncementBanner.astro';
                 </div>
             </h1>
             <p class="lead font-weight-normal">
-                A community effort to collect a curated set of analysis pipelines built using Nextflow.
+                A community effort to collect a curated set of open&#8209;source analysis pipelines built using
```
Comment (ewels) [inline sites/main-site/src/pages/index.astro:28]: Suggestion by @jfy133 
```suggestion
                A global community effort to collect a curated set of open&#8209;source analysis pipelines built using
```

CleanShot 2024-09-13 at 10 47 58@2x

**Comment by mashehu** [comment]
Stroke width of open science logo is too thin. Nice addition otherwise. Not sure if we need the sentence about maintainers on the main homepage

File: sites/main-site/src/pages/index.astro [hunk 3]
```diff
@@ -100,9 +101,14 @@ import AnnouncementBanner from '@components/navbar/AnnouncementBanner.astro';
...
-                    nf-core pipelines use GitHub releases to tag stable versions of the code and software, making
-                    pipeline runs totally reproducible.
+                <HomepageCard
+                    title="Open source"
+                    name="opensource"
+                    icon="cib:open-source-initiative"
+                    icon_class="text-success"
+                >
+                    All nf-core code is licensed under the MIT license and is available on GitHub. Pipelines can be used
+                    as they are or forked and modified as needed.
```
Comment (jfy133) [inline sites/main-site/src/pages/index.astro:111]:
```suggestion
                    All nf-core code is licensed under the MIT license and is available on GitHub. Pipelines can be used
                    modified, forked, by anyone.
```

**Review by jfy133 — COMMENTED** [review]
One other thing could be to keep the stable releaeses thing and collpase this box

image

 which I think is redundant, I don't see why cloud needs it's own special box - if I can switch between laptop/HPC/cloud easy peasy now that's the stuff
 
 Also should we update these while we are at it?
 
 
image

File: sites/main-site/src/pages/index.astro (hunk 1 above)
Comment (jfy133) [reply sites/main-site/src/pages/index.astro:138]: Maybe not just pipelines...? the project in general?

Comment (jfy133) [reply sites/main-site/src/pages/index.astro:138]: Because people can also contribute to infra/docs etc.

Comment (ewels) [reply sites/main-site/src/pages/index.astro:138]: ok, will change pipelines -> code on the line above and use @christopher-hakkaart 's text with project on this line.

Comment (ewels) [reply sites/main-site/src/pages/index.astro]: Done in 854dc9ba

**Comment by ewels** [comment]
> Stroke width of open science logo is too thin.

Updated 👍🏻

File: sites/main-site/src/pages/index.astro (hunk 3 above)
Comment (ewels) [reply sites/main-site/src/pages/index.astro:111]: Feel like we need an `and` there somewhere.. 🤔 

```suggestion
                    All nf-core code is licensed under the MIT license and is available on GitHub. Pipelines can be used
                    modified and forked, by anyone.
```

**Comment by ewels** [comment]
> I don't see why cloud needs it's own special box

Because people don't read - if I see "portable and reproducible" I don't think "this academic project will work in the cloud". I think it's fairly valuable tbh.

I collapsed the stable versions thing because I thought the same thing about that box - reproducible + CI / CD etc all implies stable releases, I kind of felt that this was a given.

Mind you, if we _do_ add a list of supporters and that includes the AWS logo then I guess that'll get the point across quite nicely as well. In which case we could keep the releases one 🤔

**Comment by ewels** [comment]
> based on the previous discussions

👉🏻  https://github.com/nf-core/website/issues/2657

**Comment by jfy133** [comment]
> Because people don't read - if I see "portable and reproducible" I don't think "this academic project will work in the cloud". I think it's fairly valuable tbh.

Mmm, fair enough but I don't think proportionally many academics are using the cloud (well in Europe anyway, I think it's different in the US), Making the Azure/AWS logo clearer (or including that in the portability box) would work indeed

**Comment by ewels** [comment]
@jfy133 ok - I had a go at reworking the boxes we discussed. Hopefully this is better?

CleanShot 2024-09-19 at 01 15 56

I'd like to leave the "supported by" stuff to another PR if that's ok.

**Comment by ewels** [comment]
Ok and I tweaked the other strip to address https://github.com/nf-core/website/issues/2657 - hopefully better now:

CleanShot 2024-09-19 at 01 26 21@2x

**Comment by d4straub** [comment]
I just stumbled upon this issue. I like the additions. But I am not sure the fancy animations in the figures are a good idea. They are disturbing me when reading/looking at it because they constantly attract attention and seem to highlight the most important points (but they are not more important than others, I think). Maybe just me who is not used to flashy internet because I block all that stuff in the browser :D

**Comment by ewels** [comment]
Yeah, you're totally right. In fact most of these icons have remained unchanged since the very start of nf-core. I've had a stab at a refresh:

CleanShot 2024-09-19 at 09 50 03@2x

Thoughts?

**Comment by ewels** [comment]
Bit more homogenous / boring with the new icons, but I think that's ok?

I tried green icons too, but was worried that it was a bit too much green:

CleanShot 2024-09-19 at 09 52 41@2x

**Comment by ewels** [comment]
meh, maybe Green is nicer. Pushing that.

**Comment by jfy133** [comment]
I left a bunch of lazy comments on slack here https://nfcore.slack.com/archives/C072TAN9RDE/p1726702199869559

**Comment by ewels** [comment]
Updated: 
CleanShot 2024-09-19 at 10 04 03@2x

**Comment by jfy133** [comment]
Is the deploy preview broken again here @ewels ? Or do I have to wait longer?

**Comment by mashehu** [comment]
Don't like that the two check marks have slightly different styles/ratio, but that's material icons 😬

**Comment by jfy133** [comment]
> Bit more homogenous / boring with the new icons, but I think that's ok?
> 
> I tried green icons too, but was worried that it was a bit too much green:
> 
> CleanShot 2024-09-19 at 09 52 41@2x

I find the green too strong/distracting from the text actually

File: sites/main-site/src/pages/index.astro [hunk 4]
```diff
@@ -88,58 +89,88 @@ import AnnouncementBanner from '@components/navbar/AnnouncementBanner.astro';
...
                     <HomepageCard
                         title="Develop with the community"
                         fontawesome="fa-duotone fa-people-carry text-secondary"
                         btn_txt="Join Slack"
                         btn_href="join/"
                     >
-                        Come and talk to us <em>before</em> you start writing a pipeline to find collaborators and check
-                        that your pipeline is suitable for nf-core.
+                        Join forces with others! We promote collaboration over duplication: before you start coding,
+                        please propose your idea in Slack..
```
Comment (jfy133) [inline sites/main-site/src/pages/index.astro:173]:
```suggestion
                        please propose your idea in Slack.
```

**Review by jfy133 — APPROVED** [review]
Small typo fix,

While we are at it:

image

I don't think the iGenomes reference makes here, and certianly by now is not the case. I would consider removing it.

**Comment by ewels** [comment]
> Is the deploy preview broken again here @ewels ? Or do I have to wait longer?

I think it breaks whenever I change any CSS. The change was actually just left over from me testing colours and unused, just removed it - hopefully the preview will work again now? 🤞🏻 



> I think I would stick with grey.

Done

**Comment by ewels** [comment]
> I don't think the iGenomes reference makes here, and certianly by now is not the case. I would consider removing it.

I look forward to when we can be proud of the references we ship again 😓 Updated 👍🏻

**Pull request merged**